bitnami/mysql/values.yaml

```
@@ -507,7 +507,7 @@ secondary:
   ## @param secondary.existingConfiguration Name of existing ConfigMap with MySQL Secondary configuration.
```

You need to update this comment too:
```suggestion
  ## @param secondary.existingConfigmap Name of existing ConfigMap with MySQL Secondary configuration.
```